server/tests/Unit/Administration/Infrastructure/Settings/Controller/PostSettingsControllerTest.php

```diff
@@ -52,6 +52,8 @@ final public function testPostSettingsSuccess(): void
     }
 
     /**
+     * @throws Exception
```

Comment:
Exception catch already all the other kind of exceptions. Maybe you should prefer more specific throws docblock

Reply:
This is this one: 
```php 
use Doctrine\DBAL\Driver\Exception;
```

Reply:
oh my bad! :)